brave/brave-core
# Change Rewards onboarding image in Japan

Resolves https://github.com/brave/brave-browser/issues/15873

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clean profile
- Enable bitflyer in brave://flags (if necessary)
- Close browser
- Launch browser with `--rewards=countryid=19024` flag
- Visit brave://rewards
- Click through onboarding and verify that new image for "Verify your wallet" page looks correct
